fables/tree.py

```diff
@@ -144,22 +144,31 @@ def __str__(self) -> str:
 class MimeTypeFileNode(FileNode):
     MIMETYPES: List[str] = []
     EXTENSIONS: List[str] = []
+    EXTENSIONS_TO_EXCLUDE: List[str] = []
 
     @classmethod
     def is_my_mimetype_or_extension(
         cls, mimetype: Optional[str], extension: Optional[str]
     ) -> bool:
-        if mimetype is not None and mimetype in cls.MIMETYPES:
-            if mimetype == cls.MIMETYPES[0]:
+        mimetype = mimetype or "_"
+        extension = extension or "_"
+        if mimetype in cls.MIMETYPES:
+            # trust the best mimetype match except when the extension is exluded
+            if (
+                mimetype == cls.MIMETYPES[0]
+                and extension not in cls.EXTENSIONS_TO_EXCLUDE
```

Comment:
Need to call `.lower()` here as well? Or line 154 `extension = (extension or "_").lower()`?

Reply:
Oh that is a nice solution! I will do that

Reply:
(and nice catch)